beacon_node/network/src/sync/network_context.rs

```diff
@@ -47,7 +47,7 @@ impl<T: BeaconChainTypes> SyncNetworkContext<T> {
     pub fn new(
         network_send: mpsc::UnboundedSender<NetworkMessage<T::EthSpec>>,
         network_globals: Arc<NetworkGlobals<T::EthSpec>>,
-        beacon_processor_send: BeaconProcessorSend<T>,
+        network_beacon_processor: Arc<NetworkBeaconProcessor<T>>,
```

Comment:
Now `SyncNetworkContext` has two `Arc`s to the `NetworkGlobals`, so the unwrapped one could be removed.. later maybe?

Reply:
Thanks @divagant-martian 🙏 Fixed in d2e5f9de7